src/test/java/org/apache/commons/lang3/ClassUtilsTest.java

```diff
@@ -88,6 +88,14 @@
             // empty
         }
     }
+    private static class ABG {
+        public void publicMethod(){}
+        private void privateMethod(){}
+        void defaultMethod(){}
+        public final void finalMethod(){}
+        public static void  staticMethod(){}
```

Comment:
Remove double whitespace after void. The same change in the javadoc of the method in ClassUtils.

Reply:
Done